knowledge-base/actions/benc-uk/action-security.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Workflow Dispatch'
```

Comment:
@vandana41 please correct the `filePath` to `knowledge-base/actions/benc-uk/workflow-dispatch/action-security.yml`